Broken Master

### *Product:* Software
      
### *Category:* Firmware
   
**Describe the issue found**

PR # 164 merge broke master
 
src/devices/spibus.c:46:12: error: conflicting types for 'spi_get_handle'
 SPI_Handle spi_get_handle(uint32_t index) {
            ^~~~~~~~~~~~~~
In file included from src/devices/spibus.c:19:0:
/home/osboxes/OpenCellular/firmware/ec/inc/common/spibus.h:31:12: note: previous declaration of 'spi_get_handle' was here
 SPI_Handle spi_get_handle(unsigned int index);
            ^~~~~~~~~~~~~~
make: *** [src/devices/spibus.o] Error 1

commit 2b8133137ac40facb84e75d9fc1335dc0a109e3e
commit 8e95b8f9d62e5b60d5a03fc55b64bee147975c49


**Please provide all necessary steps to reproduce the issue**
Build from firmware/ec

--- a/firmware/ec/inc/common/spibus.h
+++ b/firmware/ec/inc/common/spibus.h
@@ -28,7 +28,7 @@ typedef struct SPI_Dev {
 /*****************************************************************************
  *                             FUNCTION DECLARATIONS
  *****************************************************************************/
-SPI_Handle spi_get_handle(unsigned int index);
+SPI_Handle spi_get_handle(uint32_t index);
 
 ReturnStatus spi_reg_read(SPI_Handle spiHandle,
                           OcGpio_Pin *chip_select,

